static/js/popover_menus.js

```diff
@@ -87,4 +107,196 @@ export function initialize() {
+                // message in the current message list (and
+                // compose_actions.respond_to_message doesn't take a message
+                // argument).
+                compose_actions.quote_and_reply({trigger: "popover respond"});
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".reminder_button", (e) => {
+                popovers.render_actions_remind_popover(
+                    $(".selected_message .actions_hover")[0],
+                    message_id,
+                );
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".popover_toggle_collapse", (e) => {
+                if (row) {
+                    if (message.collapsed) {
+                        condense.uncollapse(row);
+                    } else {
+                        condense.collapse(row);
+                    }
+                }
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".view_edit_history", (e) => {
+                const message_history_cancel_btn = $("#message-history-cancel");
+
+                message_edit_history.show_history(message);
+                message_history_cancel_btn.trigger("focus");
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".delete_message", (e) => {
+                message_edit.delete_message(message_id);
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".popover_mute_topic", (e) => {
+                muting_ui.mute_topic(message.stream_id, message.topic);
+                e.preventDefault();
+            });
+
+            $popper.one("click", ".popover_unmute_topic", (e) => {
+                muting_ui.unmute_topic(message.stream_id, message.topic);
+                e.preventDefault();
+            });
+
+            popovers.clipboard_enable(".copy_link").on("success", (e) => {
+                // e.trigger returns the DOM element triggering the copy action
+                const message_id = e.trigger.getAttribute("data-message-id");
+                const row = $(`[zid='${CSS.escape(message_id)}']`);
+                row.find(".alert-msg")
+                    .text($t({defaultMessage: "Copied!"}))
+                    .css("display", "block")
+                    .delay(1000)
+                    .fadeOut(300);
+
+                setTimeout(() => {
+                    // The Cliboard library works by focusing to a hidden textarea.
+                    // We unfocus this so keyboard shortcuts, etc., will work again.
+                    $(":focus").trigger("blur");
+                }, 0);
+            });
+
+            $popper.one("click", ".reaction_button", (e) => {
+                e.preventDefault();
+                instance.hide();
+                setTimeout(() => {
+                    emoji_picker.toggle_emoji_popover(instance.reference.parentElement, message_id);
+                }, 0);
```

Comment:
This function seems to be new in this change; it seems like we have a similar click hander in `static/js/emoji_picker.js` that this is overriding / duplicating?  

I would sorta expect that we just need the `instance.hide` here, and that we'd let the global click handler for `.reaction_button` do its magic.

Reply:
We don't need the one in emoij_picker actually.